icu4c/source/test/cintltst/unumberformattertst.c

```diff
@@ -212,6 +216,64 @@ static void TestExampleCode() {
 }
 
 
+static void TestSimpleNumberFormatter() {
+    // This is the example in usimplenumberformatter.h
```

Comment:
optional: Consider either adding another comment for where this .h example ends, or else moving the example code into its own block or function.

Reply:
Split it into two test functions